src/inputmethodkit.cs

```diff
@@ -217,13 +223,13 @@ partial interface IMKCandidates {
 		NSAttributedString SelectedCandidateString { get; }
 
 		[Lion, Export ("candidateFrameTopLeft")]
-		NSPoint CandidateFrameTopLeft { set; }
+		CGPoint CandidateFrameTopLeft { set; }
 
 		[Export ("candidateFrame")]
-		NSRect CandidateFrame { get; }
+		CGRect CandidateFrame { get; }
 
 		[Export ("selectionKeys")]
-		NSObject [] SelectionKeys { get; set; }
+		NSNumber [] SelectionKeys { get; set; }
```

Comment:
do we know what's inside the `NSNumber` ? it sounds like a great place to use @dalexsoto new `[BindAs]` support :)